docs/TEMPLATES_API.md

```diff
@@ -0,0 +1,179 @@
+
+RenderforestClient.getTemplatesCategories({ language: 'en' })
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+- The supported language codes are: ar, de, en, es, fr, pt, ru, tr.
+
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/templates/get-templates-categories.js)
+
+
+### Get a Specific Template
+
+Retrieves a specific template.
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+RenderforestClient.getTemplate(701, { language: 'en' })
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+- The supported language codes are: ar, de, en, es, fr, pt, ru, tr.
+
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/templates/get-template.js)
+
+
+### Get Color-Presets of the Template
+
+Retrieves color-presets of the template.
+You can apply these color presets to your project to give it better and unique look.
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+RenderforestClient.getTemplateColorPresets(701)
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+- The number of color-presets is varying from template to template.
+
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/templates/get-template-color-presets.js)
+
+
+### Get Pluggable-Screens of the Template
+
+Retrieves pluggable-screens of the template.
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+RenderforestClient.getTemplatePluggableScreens(701)
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+- Only lego templates might have a pluggable-screen. 
+- The number of pluggable-screens is varying from template to template.
+Pluggable-Screens are grouped by categories.
+
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/templates/get-template-pluggable-screens.js)
+
+
+### Get Recommended-Custom-Colors of the Template
+
+Retrieves recommended-custom-colors of the template.
+You can apply these recommended custom colors to your project to give it better and unique look.
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+RenderforestClient.getTemplateRecommendedCustomColors(701)
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+- The number of recommended-custom-colors is varying from template to template.
+
+[See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/templates/get-template-recommended-custom-colors.js)
+
+
+### Get Template-Presets of the Template
+
+Retrieves template-presets of the template.
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+const payload = {
+  templateId: 701
+}
+RenderforestClient.getTemplatePresets(payload)
```

Comment:
In `getTemplatePresets` function an object with single key passed as argument, but in functions below (for example, `getTemplateSVGContent` or `getTemplateTheme`) arguments are passed without putting them into objects. I think it would be nice to keep consistency in function definitions.

Reply:
it's obvious that this is a mistake 😄